### ISSUE TITLE
Your browser was unable to load the scripts and styles of this page.

### ISSUE THREAD

**Issue by StefWeb12** [matomo-org/matomo#12812]
Hey Matomo-Team,

I have downloaded Piwik. At the start to the installation 2 Errormessages came on my Homepage.
I have disabled my AdBlocker. But this didn't helped.

1. Your browser was unable to load the scripts of this page.
2. Your browser was unable to load the style of this page.

I have uploaded the Files like it stands in the Documentation.

I don't know the Problem why this 2 Errormessages coming.

Here is a Pic of it:
[piwik_installationsfehler](https://user-images.githubusercontent.com/38916763/39528741-37e98984-4e25-11e8-865c-4d7c0a251dfc.PNG)

With frindly Greetings,
Stef

**Comment by StefWeb12** [comment]
Hey,

I have disabled my AdBlocker and started the Browser new. But there came the same as before when the AdBlocker ist on.

**Comment by Findus23** [comment]
Hi,

Can you then open the developer tools of your browser (<kbd>F12</kbd>) and check for errors in the console or failed requests in the network tab.
Maybe you need to completely reload the page (<kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>R</kbd>)

**Comment by Findus23** [comment]
Hm, can you try opening one of the three URLs directly?

Does the network tab show any reason for the failure?
Can you easily try another browser (like Chrome which shows more verbose errors)

**Comment by StefWeb12** [comment]
I have open it now via Google Chrome.

There i get more Errors. Some Scripts are bloc
![fehlerchrome](https://user-images.githubusercontent.com/38916763/39531005-7abe42d6-4e2a-11e8-8fb0-fe8ed5b9aa70.PNG)
ked by AntiVirus-Programm.

**Comment by Findus23** [comment]
Okay this means there is a software that is intercepting all of your web requests (and as it is not a https-site this is trivial) and blocking the files Matomo needs.

So to fix the issue, configure this software (most likely your antivirus) to not block the files anymore.

But I think you should better completely disable this feature as intercepting webrequests (or even encrypted ones) nearly always causes more insecurity than security.

And as you seem to be in Germany you will need to think about GDPR/DSGVO ([see here](https://matomo.org/blog/) for more information). And in my personal opinion it is impossible to comply to the law if you don't know where your visitors data is stored as it is transferred completely unencrypted without HTTPS. So you should be setting up HTTPS as soon as you can.

**Comment by va3093** [comment]
The problem for me was that I was using nginx to proxy requests to `https://mysite.blah/matomo` to matomo which was running in a docker container. However the client side calls were making requests to `https://mysite.blah/index.php?asdfasd`. As a result I the resources were not being found. I have changed my confif to instead redirect `https://matomo.mysite.blah`. Then it works

**Comment by Mogekoff** [comment]
I solved this problem by removing the line
`Header always set Content-Type "text/html; charset=utf-8"`
from my .htaccess

**Comment by endrep0** [comment]
For me this issue was happening when I moved an old Matomo installation to a box with PHP 8.
The old Matomo (3.x.x and 4.x.x) had old JShrink packages which were incompatible with PHP 8. JShrink's minify wasn't working, and this was causing javascript files to time out.

The solution was to replace the old JShrink in the installed matomo directory located at `/vendor/matomo-org/jshrink` or `/vendor/tedivm/jshrink` with the latest JShrink package downloaded from https://github.com/tedious/JShrink/releases

The page loads got a whole lot faster too after this issue got resolved.

(After upgrading to Matomo 5.x.x, replacing JShrink is no longer needed because that one has a new JShrink compatible with PHP 8.)
